osquery/CMakeLists.txt

```diff
@@ -98,6 +98,9 @@ ADD_OSQUERY_LINK_CORE("glog")
 ADD_OSQUERY_LINK_CORE("boost_system")
 ADD_OSQUERY_LINK_CORE("boost_filesystem")
 ADD_OSQUERY_LINK_CORE("boost_regex")
+ADD_OSQUERY_LINK_CORE("cryptsetup")
```

Comment:
@theopolis I get undefined references to crypt_() functions. I can provide extended output if needed.